Source/Core/DolphinQt/InfinityBase/InfinityBaseWindow.h

```diff
@@ -10,6 +10,8 @@
 
 #include "Common/CommonTypes.h"
 
+#include "Core/IOS/USB/Emulated/Infinity.h"
+
```

Comment:
Since you're forward declaring FigureUIPosition below, I don't think you need to include this.